pkgs/os-specific/linux/isgx/default.nix

```diff
@@ -11,14 +11,22 @@ stdenv.mkDerivation rec {
     hash = "sha256-zZ0FgCx63LCNmvQ909O27v/o4+93gefhgEE/oDr/bHw=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-compilation-on-5.8-kernel.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/intel/linux-sgx-driver/pull/120.diff";
```

Comment:
Please fetch commits instead of PRs.

Reply:
@SuperSandro2000  There are multiple commits in the PR. Do you mean to specify multiple `fetchpatch`?

Reply:
Oh, there is a squashed commit in master. Fixed.